packages/gatsby-remark-images/src/gatsby-node.js

```diff
@@ -52,7 +54,43 @@ exports.pluginOptionsSchema = function ({ Joi }) {
       .description(
         `Additionally generate WebP versions alongside your chosen file format. They are added as a srcset with the appropriate mimetype and will be loaded in browsers that support the format. Pass true for default support, or an object of options to specifically override those for the WebP files. For example, pass { quality: 80 } to have the WebP images be at quality level 80.`
       ),
-    tracedSVG: Joi.boolean()
+    tracedSVG: Joi.alternatives()
+      .try(
+        Joi.boolean(),
+        Joi.object({
+          turnPolicy: Joi.string()
+            .valid(
+              // this plugin also allow to use key names and not exact values
+              `TURNPOLICY_BLACK`,
+              `TURNPOLICY_WHITE`,
+              `TURNPOLICY_LEFT`,
+              `TURNPOLICY_RIGHT`,
+              `TURNPOLICY_MINORITY`,
+              `TURNPOLICY_MAJORITY`,
+              // it also allow using actual policy values
+              Potrace.TURNPOLICY_BLACK,
+              Potrace.TURNPOLICY_WHITE,
+              Potrace.TURNPOLICY_LEFT,
+              Potrace.TURNPOLICY_RIGHT,
+              Potrace.TURNPOLICY_MINORITY,
+              Potrace.TURNPOLICY_MAJORITY
```

Comment:
2 ways of setting it is covered by https://github.com/gatsbyjs/gatsby/blob/70b81a6e825c583387728c02d83a70e0d4e16072/packages/gatsby-remark-images/src/index.js#L264-L271